app/controllers/posts_controller.rb

```diff
@@ -7,7 +7,12 @@ def index
 
   def create
     @new_post = Post.create(post_params)
-    redirect_to '/posts'
+    if @new_post.save
+      redirect_to '\posts', notice: 'Post successful'
```

Comment:
should this not be `'/posts'` ? Or better yet - `posts_path`

Reply:
We've changed this to posts_path. The flash notice doesn't appear currently but I believe Fiona is working on that.